src/components/WidgetCalendar.vue

```diff
@@ -0,0 +1,156 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+
+const weeks = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const todayDate = new Date();
+const currentYear = todayDate.getFullYear();
+const currentMonth = todayDate.getMonth();
+
+const formattedMonthYear = computed(() => {
+  const options: Intl.DateTimeFormatOptions = { month: 'short', year: 'numeric' };
+  return todayDate.toLocaleDateString(undefined, options);
+});
```

Comment:
```suggestion
const formattedMonthYear = (() => {
  const options: Intl.DateTimeFormatOptions = { month: 'short', year: 'numeric' };
  return todayDate.toLocaleDateString(undefined, options);
})();
```
computedじゃなくてよさそう(computedはrefをトラッキングするときだけ)